.github/workflows/validate.yml

```diff
@@ -11,7 +11,7 @@ on:
 permissions: read-all
 
 env:
-  LINT_VERSION: v1.64.8
+  LINT_VERSION: v2.0
```

Comment:
shouldn't we pin x.y.**z** here. Otherwise the linter might can get a ptach update that randomly breaks on a PR, no?

Reply:
My thinking was by omitting the patch version we have less noise PRs to upgrade, and from my experience with other repos I don't remember that patch bump results in any new warnings.

Of course YMMV, so let me know which way you like it.

Reply:
personally I hate breakages on random PRs, in case of community contributors they might get confused and you then get to tell them to rebase, etc...

That said I agree if you have experience that it is rather stable I am fine to give that a go, my (bad) experiences about linter updates are from the fact that we had enable-all which of course causes new warnings very often but you already fixed that problem here.

Reply:
I agree that enable-all is a PITA, and this is actually my second attempt at killing it (🤞🏻 it will stay this time).